ops/docker/ci-builder/Dockerfile

```diff
@@ -11,10 +11,11 @@ RUN apt-get update && \
   chmod +x ./rustup.sh && \
   ./rustup.sh -y
 
-COPY ./.foundryrc ./.foundryrc
+COPY ./.abigenrc ./.abigenrc
```

Comment:
Is this file checked in?

Reply:
Yes, please see the pr this is stacked ontop of #6556